docs/server/source/cloud-deployment-templates/node-on-kubernetes.rst

```diff
@@ -199,8 +243,82 @@ Create the required StatefulSet using:
 You can check its status using the commands ``kubectl get statefulsets -w``
 and ``kubectl get svc -w``
 
- 
-Step 6: Run BigchainDB as a Deployment
+You may have to wait for upto 10 minutes wait for disk to be created
+and attached on the first run. The pod can fail several times with the message
+specifying that the timeout for mounting the disk has exceeded.
+
+
+Step 7: Initialize a MongoDB Replica Set - Optional
+---------------------------------------------------
+
+This step is required only if you are planning to set up multiple
+`BigchainDB nodes
+<https://docs.bigchaindb.com/en/latest/terminology.html#node>`_, else you can
+skip to the :ref:`step 9 <Step 9: Run BigchainDB as a Deployment>`.
+
+
+Login to the running MongoDB instance and access the mongo shell using:
+
+.. code:: bash
+   
+   $ kubectl exec -it mdb-0 -c mongodb -- /bin/bash
+   $ mongo --port 27017
+
+We initialize the replica set by using the ``rs.initialize()`` command from the
```

Comment:
I think you mean the `rs.initiate()` command, since that's what's in the two code blocks below.